.github/workflows/validate-projects.yml

```diff
@@ -0,0 +1,64 @@
+name: Validate, Update, and Deploy
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Nit: As a maintainer of APIs that occasionally get slammed at the top of the hour, I try to pick a slightly more arbitrary time to help offset some of the load.